fdm-app/app/types/extended-user.d.ts

```diff
@@ -2,6 +2,8 @@ import type { User as BetterAuthUser } from "better-auth"
 
 // Extend the User type from better-auth to include additional fields
 export interface ExtendedUser extends BetterAuthUser {
+    userName: string | null | undefined
     firstname?: string | null
```

Comment:
why undefined option?

Reply:
Will be resolved in 2e907009b7d5d92a37c79c6a10351a664796c784